modules/EnsEMBL/Web/Component/Gene/ComparaTree.pm

```diff
@@ -38,7 +38,7 @@ sub get_details {
   my $member = $object->get_compara_Member($cdb);
   return (undef, '<strong>Gene is not in the compara database</strong>') unless $member;
 
-  my $strain_tree  = $self->hub->species_defs->get_config($self->hub->species,'RELATED_TAXON') if($self->is_strain || $self->hub->species_defs->IS_STRAIN_OF || $self->hub->param('strain'));
+  my $strain_tree  = $self->hub->species_defs->get_config($self->hub->species,'RELATED_TAXON') if ($self->is_strain || $self->hub->param('strain'));
```

Comment:
`$hub->is_strain` ? Ignore if we are keeping the method in Component.pm

Reply:
Done.